### PR TITLE
[BXMSDOC-6632] Unmanaged immutable KIE server environment in operator

### PR DESCRIPTION
**Pull request by mramendi**

### CODE REVIEW

File: doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc [hunk 1]
```diff
@@ -21,7 +21,9 @@ In {PRODUCT} {ENTERPRISE_VERSION}, high-availability {CENTRAL} functionality dep
...
+** `rhpam-production-immutable`: An alternate environment for running existing services for staging and production purposes. You can configure one or more {KIE_SERVER} pods that build a service from source and replicate each pod as necessary. In this environment, when you deploy a {KIE_SERVER} pod, it builds an image that loads and starts a service or group of services. You cannot stop any service on the pod or add any new service to the pod. If you want to use another version of a service or modify the configuration in any other way, you deploy a new server image to replace the old one. In this system, the {KIE_SERVER} runs like any other pod on the OpenShift environment. You can use any container-based integration workflows and you do not need to use any other tools to manage the pods.
++
+Optionally, you can include {CENTRAL} Monitoring in this environment to monitor, stop, and restart the execution of services.
```
Comment (spolti) [inline doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc:26]: you can include {CENTRAL}  Monitoring  by explicit configure in in the Wizard. in this environment to monitor


execution of services.  -> Business Process.

File: doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc [hunk 2]
```diff
@@ -21,7 +21,9 @@ In {PRODUCT} {ENTERPRISE_VERSION}, high-availability {CENTRAL} functionality dep
...
+** `rhpam-production-immutable`: An alternate environment for running existing services for staging and production purposes. You can configure one or more {KIE_SERVER} pods that build a service from source and replicate each pod as necessary. In this environment, when you deploy a {KIE_SERVER} pod, it builds an image that loads and starts a service or group of services. You cannot stop any service on the pod or add any new service to the pod. If you want to use another version of a service or modify the configuration in any other way, you deploy a new server image to replace the old one. In this system, the {KIE_SERVER} runs like any other pod on the OpenShift environment. You can use any container-based integration workflows and you do not need to use any other tools to manage the pods.
```
Comment (spolti) [inline doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc:24]: Or download already build KJars from a pre configured maven repo. -> add a link to the doc where the KIE_SERVER_DEPLOYMENT_ID env is used.

Comment (spolti) [reply doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc:24]: you deploy -> you need to deploy or you have to deploy

Comment (spolti) [reply doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc:24]: In this system, the {KIE_SERVER} runs like any other pod on the OpenShift environment.
imho can be removed, it does not makes much sense to me.

File: doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc [hunk 3]
```diff
@@ -1,10 +1,19 @@
...
+After you set the basic and security configuration of a {PRODUCT} environment using {operator_name}, you can optionally configure settings for the {CENTRAL} 
+ifdef::PAM[]
+or {CENTRAL} Monitoring 
+endif::PAM[]
+component of the environment.
+
+All environment types except `{PRODUCT_INIT}-production-immutable` include this component.
 
+ifdef::PAM[]
+By default, the `{PRODUCT_INIT}-production-immutable` environment does not include {CENTRAL} Monitoring. To include {CENTRAL} Monitoring in this environment, you must set the number of replicas for the {CENTRAL} Monitoring pod in the *Replicas* field.
```
Comment (spolti) [inline doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc:13]: not only the number of replicas, but any console property like name, image, enc, etc..

**Review by spolti — CHANGES_REQUESTED** [review]

File: doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc [hunk 4]
```diff
@@ -33,8 +32,15 @@ ifdef::DM[]
...
 ====
 +
-** `rhdm-production-immutable`: An alternate environment for running existing services for staging and production purposes. You can configure one or more {KIE_SERVER} replicated pods that build a service from source. In this environment, when you deploy a {KIE_SERVER} pod, it builds an image that loads and starts a service or group of services. You cannot stop any service on the pod or add any new service to the pod. If you want to use another version of a service or modify the configuration in any other way, you deploy a new server image and displace the old one. In this system, the {KIE_SERVER} runs like any other pod on the OpenShift environment. You can use any container-based integration workflows and do not need to use any other tools to manage the pods.
 endif::DM[]
+** `{PRODUCT_INIT}-production-immutable`: An alternate environment for running existing services for staging and production purposes. You can configure one or more {KIE_SERVER} pods that build services from source or pull them from a Maven repository. You can then replicate each pod as necessary.
```
Comment (spolti) [inline doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc:36]: why always using services than business process/rules?

File: doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc [hunk 5]
```diff
@@ -33,8 +32,15 @@ ifdef::DM[]
...
 endif::DM[]
+** `{PRODUCT_INIT}-production-immutable`: An alternate environment for running existing services for staging and production purposes. You can configure one or more {KIE_SERVER} pods that build services from source or pull them from a Maven repository. You can then replicate each pod as necessary. 
++
+You cannot stop any service on the pod or add any new service to the pod. If you want to use another version of a service or modify the configuration in any other way, deploy a new server image to replace the old one. You can use any container-based integration workflows and you do not need to use any other tools to manage the pods.
++
+When configuring this environment, you must customize the {KIE_SERVER} in the configuration console and click the *Set immutable server configuration* button.
```
Comment (spolti) [inline doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc:40]: configuration console does not fits well here.;

File: doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc [hunk 6]
```diff
@@ -33,8 +32,15 @@ ifdef::DM[]
...
 endif::DM[]
+** `{PRODUCT_INIT}-production-immutable`: An alternate environment for running existing services for staging and production purposes. You can configure one or more {KIE_SERVER} pods that build services from source or pull them from a Maven repository. You can then replicate each pod as necessary. 
++
+You cannot stop any service on the pod or add any new service to the pod. If you want to use another version of a service or modify the configuration in any other way, deploy a new server image to replace the old one. You can use any container-based integration workflows and you do not need to use any other tools to manage the pods.
++
+When configuring this environment, you must customize the {KIE_SERVER} in the configuration console and click the *Set immutable server configuration* button. 
+ifdef::PAM[]
+Optionally, you can also use the configuration console to include {CENTRAL} Monitoring in this environment to monitor, stop, and restart the execution of process services.
```
Comment (spolti) [inline doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc:42]: configuration console would be the wizard ui?

**Review by spolti — CHANGES_REQUESTED** [review]

File: doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc (hunk 5 above)
Comment (mramendi) [reply doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc:40]: We don't use the term "wizard".

**Review by spolti — APPROVED** [review]

**Pull request merged**